src/services/feedProcessors/BinanceMultiProcessor.ts

```diff
@@ -48,35 +48,20 @@ export default class BinanceMultiProcessor {
     return this.sortOutput(feedFetchers, outputs);
   }
 
-  private createParams(feedFetchers: FeedFetcher[]): InputParams[] {
-    const inputs: InputParams[] = [];
-
-    feedFetchers.forEach((fetcher) => {
-      if (fetcher.name != FetcherName.BINANCE) return;
-
-      const {id, currency} = fetcher.params as InputParams;
-      inputs.push({id, currency});
-    });
-
-    return inputs;
-  }
+  private sortOutput(feedFetchers: FeedFetcher[], prices: NumberOrUndefined[]): NumberOrUndefined[] {
+    const result: NumberOrUndefined[] = [];
+    result.length = feedFetchers.length;
 
-  private sortOutput(feedFetchers: FeedFetcher[], values: OutputValues[]): number[] {
-    const inputsIndexMap: {[key: string]: number} = {};
+    let priceIx = 0;
 
     feedFetchers.forEach((fetcher, index) => {
-      const {id, currency} = fetcher.params as InputParams;
-      inputsIndexMap[`${id}:${currency}`] = index;
-    });
-
-    const result: number[] = [];
-    result.length = feedFetchers.length;
-
-    values.forEach(({id, currency, value}) => {
-      const index = inputsIndexMap[`${id}:${currency}`];
+      if (fetcher.name == FetcherName.BINANCE) {
```

Comment:
I would keep the way we do in all other places
```suggestion
      if (fetcher.name != FetcherName.BINANCE) return;
```

Reply:
done